packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -803,7 +805,34 @@ export default class Meetings extends WebexPlugin {
     this.meetingCollection.set(meeting);
 
     try {
-      await meeting.fetchMeetingInfo({});
+      // if no participant has joined the scheduled meeting (meaning meeting is not active) and we still get a locusEvent,
+      // it means the meeting will start in 5-6 min. In that case, we want to fetchMeetingInfo
+      // between 5 and 2 min (random between 3 minutes) before the meeting starts
+      // to avoid a spike in traffic to the wbxappi service
+      let waitingTime = 0;
+
+      if (destination.meeting) {
+        const {startTime} = destination.meeting;
+        const startTimeDate = new Date(startTime);
+        const startTimeDatestamp = startTimeDate.getTime();
+        const timeToStart = startTimeDatestamp - Date.now();
+        const maxWaitingTime = Math.max(Math.min(timeToStart, 3 * 60 * 1000), 0);
+
+        waitingTime = Math.round(Math.random() * maxWaitingTime);
+      }
+      let isMeetingActive = false;
+
+      if (destination.fullState) {
+        isMeetingActive = destination.fullState.active;
+      }
+      const {enableUnifiedMeetings} = this.config.experimental;
+
+      if (enableUnifiedMeetings && !isMeetingActive && useRandomDelayForInfo && waitingTime > 0) {
+        meeting.setTimeoutFetchMeetingInfoId = setTimeout(() => meeting.fetchMeetingInfo({}), waitingTime);
```

Comment:
I think here was also need to call `meeting.parseMeetingInfo()` so that some basic fields like locusUrl, sipUri get populated (point 2 from the jira description)